examples/set-default-labels-nested-package/app1_ghost/resources.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ghost
+spec:
+  replicas: 2
+  template:
+    spec:
+      containers:
+        - name: ghost
+          image: ghost:3-alpine
+          ports:
+            - containerPort: 2368
+          env:
+            - name: url
+              value: http://localhost:8080
+            - name: database__connection__database
```

Comment:
It doesn't match for this example. But it feels strange to see that it uses 2 underscore (`__`) here.

Reply:
fixed. I was just grabbing an example some where, didn't expect the content is valid. Obviously good engineers care that 😆